Update to 1.2.6.1

ryanvolz opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

ryanvolz committed:
Update to 1.2.6.1

ryanvolz committed:
Add ryanvolz to maintainers.

ryanvolz committed:
MNT: Re-rendered with conda-build 3.21.7, conda-smithy 3.15.1, and conda-forge-pinning 2021.12.11.20.48.53

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ryanvolz committed:
Switch to github source url so autotick bot gets updates.

With alsa's irregular version numbers (sometimes 4 parts, sometimes 3),
having the autotick bot watch the github tag feed seems to be the only
feasible way to get reliable automatic version updates.

ocefpaf commented on recipe/meta.yaml:
```diff
@@ -40,3 +40,4 @@ extra:
   recipe-maintainers:
     - scopatz
     - ocefpaf
+    - ryanvolz
```
:heart:
